lib/index.ts

```diff
@@ -105,9 +105,13 @@ export class HUPActionHelper {
 		const actionName = `resinhup${fromMajor}${toMajor}` as ActionName;
 
 		const { actionsConfig } = this;
-		const deviceSpecific = actionsConfig.deviceTypes[deviceType];
+		const allDevices = actionsConfig.deviceTypesDefaults;
+		const deviceSpecific = actionsConfig.deviceTypes[deviceType] || {};
 
-		if (deviceSpecific == null || deviceSpecific[actionName] == null) {
+		if (
+			allDevices[actionName] == null &&
+			(deviceSpecific == null || deviceSpecific[actionName] == null)
```

Comment:
Since we are doing `actionsConfig.deviceTypes[deviceType] || {}`, I think that this can be simply
```suggestion
		deviceSpecific[actionName] == null
```

Reply:
Done